//test error

int foo() {
    return main();
}

int main() {
    return 42;
}

